src/main.cpp

```diff
@@ -2383,6 +2393,27 @@ void loopLED() {
     }
 }
 
+void loopWATER() {
+        if (digitalRead(PIN_WATERSENSOR) == HIGH) {
+            //empty
+            if (waterFull == true)
+            {
+                debugPrintf("Water empty\n");
```

Comment:
logic error waterFull puts out water empty

Reply:
No its only printing if the "old state" is full, and the "new state" is empty. So the if statement ensures that only state changes trigger a print